src/QandaAtom.tsx

```diff
@@ -1,7 +1,445 @@
 import React from 'react';
-
 import { QandaAtomType } from './types';
+import { css } from 'emotion';
+import { neutral, error } from '@guardian/src-foundations/palette';
+import { headline, textSans, body } from '@guardian/src-foundations/typography';
+
+const { useState } = React;
+
+// CSS
+const detailStyling = css`
+    margin: 16px 0 36px;
+    background: ${neutral[93]};
+    padding: 0 5px 6px;
+    border-image: repeating-linear-gradient(
+            to bottom,
+            ${neutral[86]},
+            ${neutral[86]} 1px,
+            transparent 1px,
+            transparent 4px
+        )
+        13;
+    border-top: 13px solid black;
+    border-left: 0;
+    border-right: 0;
+    border-bottom: 0;
+    position: relative;
+    summary {
+        list-style: none;
+        margin: 0 0 16px;
+    }
+    summary::-webkit-details-marker {
+        display: none;
+    }
+
+    @media (min-width: 46.25em) {
+        ul {
+            list-style: none;
+            margin-bottom: 1rem;
+        }
+    }
+
+    ul {
+        list-style: none;
+        margin: 0 0 0.75rem;
+        padding: 0;
+    }
+
+    li {
+        margin-bottom: 0.375rem;
+        padding-left: 1.25rem;
+    }
+
+    li:before {
```

Comment:
Hmm, we seem to be duplicating this into here, what we're doing in dotcom rendering. not sure what the solution is off the top of my head!